api/permissions_handlers_test.go

```diff
@@ -26,7 +26,7 @@ func beforeEachPermissionsHandlers(t *testing.T) {
 
 func TestPermissionsDeleteHandlerNonExistentID(t *testing.T) {
 	beforeEachRolesHandlers(t)
-	rootSA := helpers.CreateRootServiceAccountWithKeyPair(t)
+	rootSA := helpers.CreateRootServiceAccountWithKeyPair(t, "user", "user@test.com")
```

Comment:
Suggestion: Maybe add a `helpers.DefaultRootKeyPairServiceAccount(t)` that returns an account with a preset name/email?